### PR TITLE
Add RBAC Guidelines

### PR DESCRIPTION
**Pull request by agrimmer**

### CODE REVIEW

**Comment by jetzlstorfer** [comment]
Thanks for the PR! 
Does that mean all existing keptn-sandbox projects would have to be updated once the RBAC rules are effective, e.g, with the next keptn release? Or is the access to API tokens (e.g., unleash or dynatrace) still working?

**Comment by agrimmer** [comment]
The currently used `default` ServiceAcount will still be available, however, with a limited set of rights. If you for example access the API token using the k8s-API, a dedicated RBAC rule is required.

File: README.md [hunk 1]
```diff
@@ -29,9 +29,95 @@ For existing repositories in the Keptn Sandbox Organization the respective code
...
+
+## RBAC Guidelines
+* Use the Service Account `keptn-default` of the keptn installation namespace wherever possible. This account has no k8s permissions assigned.
+* Never apply additional Roles or Rolebindings on the Service Account `keptn-default`
+* If your service needs permissions on k8s, create a new service account with the name `keptn-<your-service-name>` in the keptn installation namespace
+* Apply Roles, ClusterRoles, Rolebindings and ClusterRoleBindings with a minimum set of useful permissions on that service account
+* Prefer namespaced Roles and Rolebindings over ClusterRoles and ClusterRoleBindings
+
+### Example:
+Let's image, the service (deployment) `keptn-sample-service` needs read access for the secret "keptn-sample-secret".
```
Comment (jetzlstorfer) [inline README.md]: we need to clarify this sentence, e.g.,:
Let's imagine the Keptn service (which corresponds to a Kubernetes deployment and service) named `keptn-sample-service` requires read-access for a secret called `keptn-sample-secret` in the `keptn` namespace.

**Review by jetzlstorfer — CHANGES_REQUESTED** [review]
We should also add the changes that are necessary if no additional permissions are needed. 

It would be helpful to have two scenarios:
Scenario 1: no additional permissions required --> what to change
Scenario 2: permissions to read a secret required --> what to change

File: README.md (hunk 1 above)
Comment (agrimmer) [reply README.md]: Thanks, addressed.

**Review by jetzlstorfer — APPROVED** [review]
lgtm

**Pull request merged**